src/steps/account/account-delete.ts

```diff
@@ -15,13 +15,24 @@ export class DeleteAccount extends BaseStep implements StepInterface {
     type: FieldDefinition.Type.ANYSCALAR,
     description: 'the value of the field',
   }];
+  protected expectedRecords: ExpectedRecord[] = [{
+    id: 'account',
+    type: RecordDefinition.Type.KEYVALUE,
+    fields: [{
+      field: 'Id',
+      type: FieldDefinition.Type.STRING,
+      description: "Account's SalesForce ID",
+    }],
+    dynamicFields: false,
+  }];
 
   async executeStep(step: Step): Promise<RunStepResponse> {
     const stepData: any = step.getData().toJavaScript();
 
     try {
       const result = await this.client.deleteAccountByIdentifier(stepData.field, stepData.identifier);
-      return this.pass('Successfully deleted Account with %s %s', [stepData.field, stepData.identifier]);
+      const record = this.keyValue('account', 'Deleted Account', result);
```

Comment:
- [ ] Returned record shouldn't include metadata about the API response that is unrelated to the account itself:

<img width="491" alt="Screen Shot 2020-02-16 at 5 21 17 PM" src="https://user-images.githubusercontent.com/3496491/74617209-e00bfd80-50e0-11ea-9df7-92114c240e65.png">